### ISSUE TITLE
remove console.log

### ISSUE THREAD

**Issue by anton6** [borisenkoandrei/year-picker#1]
Hi! Thanks for the year-picker.

Any chance we can remove console.logs()?

Example: "Справа по центру" etc

Thanks

**Comment by aleksey-pro** [comment]
когда консоль уберешь?))

**Comment by FlorianDeParis** [comment]
yeah dude, like Anton6 says, i've a console log with "Сверху  по центру" and "Снизу по центру" returned. By the way, good job dude ! ;)
